DirectEmployers/MyJobs
# [PD-3686] update some resource urls for microsites

## What's this PR do?
Switch the url for some frequently requested resources from the old (`d2e48ltfsb5exy`) to new (`d1g1peq7ldieal`), this time around we are updating these 6 libs/files:

- /framework/v2/js/code/jquery-1.7.1.js
- /framework/v2/js/code/jquery-ui-1.8.17.custom.min.js
- /dseo_static/def.ui.microsite.141-20.js
- /dseo_static/underscore-1.3.1.min.js
- /dseo_static/jquery.placeholder.js
- /dseo_static/pager.164-21.js

## Screenshots (if appropriate)
Before on microsites: 

![image](https://user-images.githubusercontent.com/27963853/30876969-2a4d5c66-a2c6-11e7-8993-d9a8041b7a2e.png)

Desired behavior:

![image](https://user-images.githubusercontent.com/27963853/30876987-3b65f85a-a2c6-11e7-94d8-31f87153942d.png)

## Where should the reviewer start?

There are 3 files that had reference to files listed above:
static/filter_carousel.js
templates/myblocks/myblocks_base.html
templates/seo_base.html
For some urls, it's direct substitution, for the ones that relied on `STATIC_URL`, I added a new settings option called `STATIC_URL_PRIMARY`, and exposed it to templates through a context processor. This required updating some settings files, are those acceptable?

## How should this be manually tested?
run microsites locally, inspect page source after adding and removing `STATIC_URL_PRIMARY` from settings. 

For example, without changing the settings file, you should see this:
![image](https://user-images.githubusercontent.com/27963853/30876913-04c96cbe-a2c6-11e7-8ce8-7fbb1029bd4d.png)


## Any background context you want to provide?

## PR Checklist

### Front-End


### Separation of Concerns
